Vision sdk 0.11.0 docs (#1215)
mapbox/android-docs: src/pages/vision/overview/ar.md

/>
}}

> By default green AR navigation lane is used to display the route. Some customization is possible. Use `VisionArView.setLaneVisualParams` to change color, width and arrow length. 

Reviewer: ```suggestion
By default green AR navigation lane is used to display the route. Some customization is possible. Use `VisionArView.setLaneVisualParams` to change color, width, and arrow length. 
```

Author: Fixed.

> Additionally AR fences could be enabled to display maneuvers with `VisionArView.setFenceVisible` method. Use `VisionArView.setFenceVisualParams` to change color, horizontal / vertical offsets and section number. 

Reviewer: Is this still correct if you rewrite:

```suggestion
To display maneuvers, enable AR fences using the `VisionArView.setFenceVisible` method. Use `VisionArView.setFenceVisualParams` to change color, horizontal and vertical offsets, and section number. 
```

Author: Fixed.
